src/sentry/api/serializers/base.py

```diff
@@ -35,8 +41,10 @@ def wrapped(cls):
 
 
 class Serializer(object):
-    def __call__(self, *args, **kwargs):
-        return self.serialize(*args, **kwargs)
+    def __call__(self, obj, attrs, user):
+        if not obj:
```

Comment:
Oh, so you're just suggesting to use `is None` instead?

I repeated this specifically because `serialize` already does:

```python
if not objects:
    return objects
```

But I guess a single item would only be applicable if it were `None`.